storm-core/src/clj/backtype/storm/ui/core.clj

```diff
@@ -510,7 +501,7 @@
                              (map #(.get_num_executors ^TopologySummary %))
                              (reduce +))]
        {"user" user
-        "stormVersion" (read-storm-version)
+        "stormVersion" (str (VersionInfo/getVersion))
```

Comment:
Filed a follow up JIAR to expose more version information on UI:
https://issues.apache.org/jira/browse/STORM-621